account_payment_order_validation_move_line/models/account_payment_mode.py

```diff
@@ -0,0 +1,11 @@
+# © 2015-2016 Akretion (<https://www.akretion.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class AccountPaymentMode(models.Model):
+
+    _inherit = "account.payment.mode"
+
+    default_selected_for_payment_id = fields.Boolean(string="To Pay Filter")
```

Comment:
rename field to `default_selected_for_payment`, by convention in Odoo, the _id prefix is kind of reserved for many2one